styleguide/derek/templates/thought-leadership-overview/_partials/about-author.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="rsTl-author-section">
+  <h2 class="rsTl-author-title">About the Author</h2>
+  <div class="rsTl-author-sectionInner">
+    <div class="rsTl-author-photo" style="background-image:url('imgs/derek/speaker-thumb.jpg');"></div>
```

Comment:
Should we have a default image in zoolander if there isn't one for some reason? This might be a question for @mikerastiello.

Reply:
Mike said we don't need to worry about this.  Marking as resolved.